### PR TITLE
Slovak (slovenčina) - added new language

### PR DESCRIPTION
**Pull request by tiborepcek**
Added `sk` folder with 7 php files to `/app/i18n`. Is that all, or should i do something more?

### CODE REVIEW

**Comment by Frenzie** [comment]
Did you base it on the dev branch or master? There might be a few new strings.

**Comment by tiborepcek** [comment]
I forked right from https://github.com/FreshRSS/FreshRSS, so it based on master, i guess.

**Comment by Frenzie** [comment]
Still a syntax error:

```
$ if [ -s php-l-results ]; then cat php-l-results; exit 1; fi

PHP Parse error:  syntax error, unexpected 'firefox' (T_STRING), expecting ')' in ./app/i18n/sk/sub.php on line 73

xargs: php: exited with status 255; aborting
```

**Comment by tiborepcek** [comment]
Thank you for your help. :)

**Comment by Frenzie** [comment]
@tiborepcek Now we have all green checkmarks. ;-)

One minor remaining issue as I indicated in my first reply. Let's see if I can be of some help with that too. I'll get back on that soon.
```
sk    - Translation is  91.3% complete.
```

**Comment by Frenzie** [comment]
Okay, these keys would ideally still be translated:

```
Untranslated key admin.check_install.tables.nok - There are one or more missing tables in the database.
Untranslated key admin.check_install.tables.ok - The appropriate tables exist in the database.
Missing key conf.display.icon.display_authors
Untranslated key sub.api.title - API
Untranslated key sub.import_export.title - Import / export
Untranslated key sub.menu.import_export - Import / export
Untranslated key gen.date.apr - Apr.
Untranslated key gen.date.Aug - \A\u\g\u\s\t
Untranslated key gen.date.aug - Aug.
Untranslated key gen.date.august - August
Untranslated key gen.date.Dec - \D\e\c\e\m\b\e\r
Untranslated key gen.date.dec - Dec.
Untranslated key gen.date.december - December
Untranslated key gen.date.feb - Feb.
Untranslated key gen.date.format_date - %s j\<\s\u\p\>S\<\/\s\u\p\> Y
Untranslated key gen.date.format_date_hour - %s j\<\s\u\p\>S\<\/\s\u\p\> Y \a\t H\:i
Untranslated key gen.date.jan - Jan.
Untranslated key gen.date.mar - Mar.
Untranslated key gen.date.Nov - \N\o\v\e\m\b\e\r
Untranslated key gen.date.nov - Nov.
Untranslated key gen.date.november - November
Untranslated key gen.date.Sep - \S\e\p\t\e\m\b\e\r
Untranslated key gen.date.sep - Sept.
Untranslated key gen.date.september - September
```

**Comment by Frenzie** [comment]
Btw @tiborepcek please check that my ignore file doesn't incorrectly ignore anything.

**Comment by tiborepcek** [comment]
> Okay, these keys would ideally still be translated: ...

All that lines are correct in slovak language, even thought they are not touched. :)

> Btw @tiborepcek please check that my ignore file doesn't incorrectly ignore anything.

As far, as i can say, it is OK. :)

File: app/i18n/sk/admin.php [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+			'nok' => 'Nepodarilo sa nájsť PDO alebo niektorý z podporovaných ovládačov (pdo_mysql, pdo_sqlite, pdo_pgsql).',
+			'ok' => 'Našiel sa PDO a aspoň jeden z podporovaných ovládačov (pdo_mysql, pdo_sqlite, pdo_pgsql).',
+		),
+		'php' => array(
+			'_' => 'Inštalácia PHP',
+			'nok' => 'Vaša verzia PHP je %s, ale FreshRSS vyžaduje minimálne verziu %s.',
+			'ok' => 'Vaša verzia PHP %s je kompatibilná s FreshRSS.',
+		),
+		'tables' => array(
+			'nok' => 'There are one or more missing tables in the database.',
```
Comment (Frenzie) [inline app/i18n/sk/admin.php]: Pssht ;-)

File: app/i18n/sk/conf.php [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+		'optimize_help' => 'Občas vykonajte na zmenšenie veľkosti databázy',
+		'purge_now' => 'Vyčistiť teraz',
+		'title' => 'Archivovanie',
+		'ttl' => 'Neaktualizovať častejšie ako',
+	),
+	'display' => array(
+		'_' => 'Zobrazenie',
+		'icon' => array(
+			'bottom_line' => 'Spodný riadok',
+			'display_authors' => 'Authors',  //TODO - Translation
```
Comment (Frenzie) [inline app/i18n/sk/conf.php]: This line was recently added.

**Review by Frenzie — APPROVED** [review]
Excellent, this should take us to 100 %. I'll wait for the tests to finish before merging. Thanks!

**Comment by tiborepcek** [comment]
So i dont need to create new PR?

**Comment by Alkarex** [comment]
@tiborepcek Thanks, and please add yourself to https://github.com/FreshRSS/FreshRSS/blob/dev/CREDITS.md

**Comment by Frenzie** [comment]
@tiborepcek
> So i dont need to create new PR?

Only very rarely. The usual method, at least on GH, is that you open a PR with your proposal, that's reviewed, you might have to make some changes, and then it's (hopefully) merged.

**Pull request merged**